services/manuals-frontend/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3.7'
+
+x-manuals-frontend: &manuals-frontend
+  build:
+    context: .
+    dockerfile: services/manuals-frontend/Dockerfile
+  image: manuals-frontend
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+  working_dir: /govuk/manuals-frontend
+
+services:
+  manuals-frontend-lite:
+    <<: *manuals-frontend
+    privileged: true
+    volumes:
```

Comment:
I don't think you need this, because the volumes are already included in `manuals-frontend` which the `lite` image inherits from.

Reply:
You've got to be quick on these, I've rebased 3 times in the last 5 minutes 😂

Reply:
I saw that! Had to keep clicking refresh!

Reply:
Should be fixed up now @leenagupte 👍